### PR TITLE
Fix   our works (div "works")

### PR DESCRIPTION
**Pull request by Ostafii4uk**

### CODE REVIEW

File: assets/CSS/style.css [hunk 1]
```diff
@@ -2,7 +2,6 @@ body{
     margin: auto;
     color: #fff;
     width: 100%;
```
Comment (ost-ptk) [inline assets/CSS/style.css]: this is a default value, no need to write it
```suggestion
```

it's all about block elements

File: assets/CSS/style.css [hunk 2]
```diff
@@ -425,25 +426,55 @@ a{
...
-.imgworks{
-    width: 100%;
-    height: 200px;
+@media (max-width: 1650px) {
+    .mainworks-item{
+        width: 250px;
+    }
 }
 
+@media (max-width: 1550px) {
```
Comment (ost-ptk) [inline assets/CSS/style.css]: your media query didn`t work right
image

File: assets/CSS/style.css [hunk 3]
```diff
@@ -457,7 +488,7 @@ a{
     height: 499px;
```
Comment (ost-ptk) [inline assets/CSS/style.css]: why 499? no magic numbers

File: index.html [hunk 4]
```diff
@@ -230,23 +230,47 @@ <h3 class="teamsubtitle">Lorem ipsum dolor sit amet proin gravida nibh vel velit
...
             <div class="mainworks">
-                    <div class="itmnw1">
-                        <img class="imgworks" src="../Tajam/assets/images/Works/1.png" alt="1">
-                        <img class="imgworks" src="../Tajam/assets/images/Works/2.png" alt="2">
-                        <img class="imgworks" src="../Tajam/assets/images/Works/3.png" alt="3">
-                        <img class="imgworks" src="../Tajam/assets/images/Works/4.png" alt="4">
-                        <img class="imgworks" src="../Tajam/assets/images/Works/5.png" alt="5">
-                        <img class="imgworks" src="../Tajam/assets/images/Works/6.png" alt="6">
-                    </div>
+                <div class="mainworks-top">
```
Comment (ost-ptk) [inline index.html:232]: better will be using one flex container for all images

File: index.html [hunk 5]
```diff
@@ -230,23 +230,47 @@ <h3 class="teamsubtitle">Lorem ipsum dolor sit amet proin gravida nibh vel velit
             </div>  
             
             <div class="mainworks">
```
Comment (ost-ptk) [inline index.html]: rename class name like 
```suggestion
            <div class="main-works">
```

**Review by ost-ptk — CHANGES_REQUESTED** [review]

**Comment by ost-ptk** [comment]
use the font abbreviation instead of font-family, font-size, font-weight, etc.

File: index.html (hunk 4 above)
Comment (Ostafii4uk) [reply index.html:232]: When one container is used, the distance between the photos is available. margin or padding does not help. And when the flex-wrap function is turned on: wrap photos become dynamic and change their position, and I need them to be static.

File: assets/CSS/style.css (hunk 3 above)
Comment (Ostafii4uk) [reply assets/CSS/style.css]: :D

**Pull request merged**